static/app/components/events/interfaces/frame/defaultTitle/index.tsx

```diff
@@ -42,6 +42,16 @@ function DefaultTitle({
   const framePlatform = getPlatform(frame.platform, platform);
   const tooltipDelay = isHoverPreviewed ? SLOW_TOOLTIP_DELAY : undefined;
 
+  /**
+   * In some cases (e.g. frame file names from SvelteKit stack frames),`frame.filename` can be
+   * prefixed with parent directory hops (e.g. `../../src/routes/....`).
+   * This function strips these hops so that users can clearly see where this file
+   * is located relative to their project root.
+   */
+  const stripRelativePathPrefix = (filename: string): string => {
+    return filename.replace(/^(..\/)*/, '');
```

Comment:
maybe escape the periods?
```suggestion
    return filename.replace(/^(\.\.\/)*/, '');
```

Reply:
oof good catch, totally missed this, thanks :D